playbooks/init/main.yml

```diff
@@ -22,6 +22,8 @@
 
 - import_playbook: basic_facts.yml
 
+- import_playbook: version.yml
```

Comment:
This change conflicts with the need to run cluster_facts.yml in order to set current version to feed into version role.

Here's where we are now:  Our openshift_repos role has become a "openshift and rhel-server repos" role which it was never intended to be.  Thus, there is a requirement to run repos before base_packages for some of our users (openstack, possibly aws provisioning plays).

What we can do instead:

In 3.11, we put a default value for openshift_release in the openshift_version role.  We can pull openshift_version in as a meta-dependency on the repos role, and use variable openshift_release in place of variable openshift_version in the repos role.

Reply:
@michaelgugino Thanks for the review! What default value would you suggest for openshift_release? Would you like me to attempt this change, or would you prefer someone with more experience with the repos roles?

Reply:
@tzumainn If you include openshift_version role as a dependency in roles/openshift_repos/meta/main.yml, you won't need to set openshift_release because we have a default value for it defined in the openshift_version role.